lib/vintage_net/connectivity/internet_checker.ex

```diff
@@ -9,15 +9,23 @@ defmodule VintageNet.Connectivity.InternetChecker do
   """
   use GenServer
 
-  alias VintageNet.Connectivity.{CheckLogic, HostList, Inspector, TCPPing}
+  alias VintageNet.Connectivity.{
+    CheckLogic,
+    HostList,
+    Inspector,
+    SSLConnect,
+    TCPPing
+  }
```

Comment:
```suggestion
  alias VintageNet.Connectivity.CheckLogic
  alias VintageNet.Connectivity.HostList
  alias VintageNet.Connectivity.Inspector
  alias VintageNet.Connectivity.SSLConnect
  alias VintageNet.Connectivity.TCPPing
```

"New" style is to just list them out.